res/values-tl/strings.xml

```diff
@@ -141,17 +141,17 @@
     <string name="Blobihilation">"Blobihilation:"</string>
     <string name="Blobageddon">"Blobageddon:"</string>
     <string name="Blobocalypse">"Blobocalypse:"</string>
-    <string name="Experienced_Nebulator">"Experienced Nebulator:"</string>
-    <string name="Expert_Nebulator">"Expert Nebulator:"</string>
+    <string name="Experienced_Nebulator">"Nakaranas Ng Nebulator:"</string>
+    <string name="Expert_Nebulator">"Dalubhasang Nebulator:"</string>
```

Comment:
Lines 144 and 145:

I don't know why it was been included a colon (represented by `:`), but please remove it.